tutorials/hosted_phoenix/hosted_phoenix_llamaindex_tutorial.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    phoenix_df = px_client.get_spans_dataframe()
</pre><p>the name here is a bit confusing. Why not spans_df?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>